handlers/main.yml

```
@@ -0,0 +1,4 @@
+- name: Mount all fstab entries `mount -a`
+  shell: mount -a
+  listen: fstab_update#
```

I don't think this `#` is intended
```suggestion
  listen: fstab_update
```